<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindowBase</class>
 <widget class="QMainWindow" name="QMainWindowBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>408</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QMotion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qmotion.qrc">
    <normaloff>:/qmotion.svg</normaloff>:/qmotion.svg</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QGroupBox" name="groupBox_webcam">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Webcam</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_size">
             <property name="text">
              <string>size : 0x0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="limit_fps">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> fps</string>
             </property>
             <property name="prefix">
              <string>limit to </string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
             <property name="value">
              <number>25</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="font">
                <font>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="toolTip">
                <string>frames per second</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="digitCount">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="fps">
               <property name="toolTip">
                <string>frames per second</string>
               </property>
               <property name="text">
                <string>fps</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="flip">
           <property name="text">
            <string>&amp;Flip</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_motion">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>160</height>
         </size>
        </property>
        <property name="title">
         <string>Motion detection parameter</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_threshold">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Try lower values to see effects</string>
               </property>
               <property name="text">
                <string>Threshold</string>
               </property>
               <property name="buddy">
                <cstring>threshold</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="threshold">
               <property name="toolTip">
                <string>Try lower values to see effects</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_color">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QFrame" name="frame_color">
               <property name="minimumSize">
                <size>
                 <width>27</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="global_marker">
             <property name="text">
              <string>Show global marker</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="component_markers">
             <property name="text">
              <string>Show component markers</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>10</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>186</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="qmotion.qrc">:/qmotion_hibou.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_surveillance">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>108</height>
         </size>
        </property>
        <property name="title">
         <string>Surveillance</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBox_save">
           <property name="toolTip">
            <string>Please select a directory in the menu</string>
           </property>
           <property name="text">
            <string>Save picture upon motion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_ftp">
           <property name="toolTip">
            <string>Please select a directory in the menu</string>
           </property>
           <property name="text">
            <string>FTP upload upon motion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_mail">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_disable_display">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Disable display (saves CPU)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_timestamp">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add timestamp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Video</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_video">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>344</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string>webcam</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Motion detection</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_motion">
           <property name="minimumSize">
            <size>
             <width>344</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string>motion</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Directory_settings"/>
    <addaction name="action_FTP_settings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_QMotion"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout_QMotion">
   <property name="icon">
    <iconset resource="qmotion.qrc">
     <normaloff>:/qmotion.svg</normaloff>:/qmotion.svg</iconset>
   </property>
   <property name="text">
    <string>About QMotion</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="icon">
    <iconset resource="qmotion.qrc">
     <normaloff>:/qt-logo-small.png</normaloff>:/qt-logo-small.png</iconset>
   </property>
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="action_Directory_settings">
   <property name="icon">
    <iconset resource="qmotion.qrc">
     <normaloff>:/recorder_folder.svg</normaloff>:/recorder_folder.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Directory settings</string>
   </property>
  </action>
  <action name="action_FTP_settings">
   <property name="icon">
    <iconset resource="qmotion.qrc">
     <normaloff>:/ftp.png</normaloff>:/ftp.png</iconset>
   </property>
   <property name="text">
    <string>FTP settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qmotion.qrc"/>
 </resources>
 <connections/>
</ui>
